node/node_v2.go

```diff
@@ -104,6 +102,43 @@ func (n *Node) DetermineChunkLocations(
 	return downloadSizeInBytes, relayRequests, nil
 }
 
+// Given a list of chunk indices we want to download, create a list of relay requests by range.
+// Although indices may not be contiguous, it is safe to assume that they will be "mostly contiguous".
+// In practice, we should expect to see at most one continuous range of indices per quorum.
+//
+// Eventually, the assignment logic ought to be refactored to return ranges of chunks instead of individual
```

Comment:
"instead of individual of individual indices"

repeat words

Reply:
fixed